addons/calendar/tests/test_event_notifications.py

```diff
@@ -30,6 +30,19 @@ def test_message_invite(self):
         }):
             self.event.partner_ids = self.partner
 
+    def test_message_invite_allday(self):
+        with self.mock_mail_gateway(), self.mock_bus(), self.mock_mail_app():
+            self.env['calendar.event'].create({
+                'name': 'Meeting',
+                'allday': True,
+                'start_date': fields.Date.today() + relativedelta(days=7),
+                'stop_date': fields.Date.today() + relativedelta(days=8),
+                'partner_ids': [(4, self.partner.id)],
+            })
+        notifications = self._new_msgs.filtered(lambda msg: msg.message_type == 'user_notification')
```

Comment:
I'm pretty sure you're not supposed to use those mocks like this (the underscore of `sef._new_msgs` is hinting that something is wrong). I don't remember well what are all mail assertions and how they work but you should take a look at `assertMailNotifications` maybe ?

Reply:
Indeed, I just needed to add `mail_create_nolog` to remove unwanted notifications.